pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/extension/PythonVersion.java

```diff
@@ -21,6 +21,10 @@
 
     private final String version;
 
+    public static String[] whitelistedPythonVersions = { "2.6", "2.7", "3.4", "3.5", "3.6" };
+    public static String defaultPython2 = "2.6";
+    public static String defaultPython3 = "3.5";
```

Comment:
I'd been thinking about these as site-wide defaults.  The use case is more like "Organization X only allows Python 2.7 and 3.6" so it would set the global defaults for that.  I wasn't thinking that these would be overridable per build.  Perhaps ideally that would come out of a site-wide configuration file, but I don't know whether that's supported (and might be too out of scope for this particular PR).

Maybe that's not the right way to think about it though?

`PythonVersion.java` seemed like a natural place to put it with that in mind, but I'll poke around and see if there's anything that stands out as possibly better.